.vale/styles/RedHat/Usage.yml

```diff
@@ -15,6 +15,7 @@ tokens:
   - alpha
   - alphabetic
   - alphabetical
+  - Application Load Balancer
```

Comment:
For example, if we merged this, Vale would flag an error for "Application Load Balancer"

Reply:
Ah, yes, thanks. Updated.